picard/coverart/providers/caa.py

```diff
@@ -75,12 +75,17 @@
 CaaSizeItem = namedtuple('CaaSizeItem', ['thumbnail', 'label'])
 
 _CAA_THUMBNAIL_SIZE_MAP = OrderedDict([
-    (250, CaaSizeItem('small', N_('250 px'))),
-    (500, CaaSizeItem('large', N_('500 px'))),
+    (250, CaaSizeItem('250', N_('250 px'))),
+    (500, CaaSizeItem('500', N_('500 px'))),
     (1200, CaaSizeItem('1200', N_('1200 px'))),
     (-1, CaaSizeItem(None, N_('Full size'))),
 ])
-
+_CAA_THUMBNAIL_SIZE_ALIASES = {
+    'large': '500',
+    'small': '250',
+    '500': 'large',
+    '250': 'small',
```

Comment:
Why does it match in two directions? As I understand it this is only needed for translating the pixel sizes to the name used in the JSON response, so we don't have a need for the reverse.

Also I wonder if we really need this. Having the name a part of `CaaSizeItem` would be totally sufficient I think and better keep things together. If we also want to have the actual size as part of `CaaSizeItem` we should probably add both?

Reply:
`CaaSizeItem` is meant for display in interface (size selector in CAA options), and should have unique items, for size selection. Perhaps we can merge and simplify, but it is likely to imply changes in UI code, which I wanted to avoid in this PR.

I added both ("large" & "500" + "small" & "250") during tests devel.
And yes, now, "large" and "small" keys are useless, removed in ff006f1.

The "500"/"250" without "large"/"small" shouldn't happen, but the reverse does (even though it is rare), see my comment below for an example.